spec/ace.md

```diff
@@ -383,6 +383,33 @@ Small quantities can be represented directly as decimals (e.g., 0.3), or using m
 
 **NOTE**: Network limits MUST NOT apply to localhost communication between apps in a pod.
 
+### Environment Isolators
+
+An application or a group of applications may often require some tweaking to the environment where they run, in order to perform in an optimal way.
+Environment isolators take care of setting up the runtime environment as needed.
+
+#### environment/sysctl
```

Comment:
I checked before venturing here, and at least FreeBSD has sysctl tuning support: https://www.freebsd.org/doc/handbook/configtuning-sysctl.html